product_docs/docs/sql_assess/1/using.mdx

```diff
@@ -0,0 +1,260 @@
+
+    ```
+     begin procin (:1 ); end;
+    ```
+
+    Is converted to the following call:
+
+    ```
+    {CALL procin (? ) }
+    ```
+
+### Command options
+
+```shell
+/usr/edb/sqlassess/bin/runSQLAssess.sh -help
+usage: assessSQL <options>
+ -appname <name>             Specify the name of the application
+ -help                       Print all available options
+ -includeUIDs <userIDs>      Specify comma separated list of User IDs for
+                             assessment consideration
+ -inputfilepath <path>       Specify the path containing the input file/s
+ -inputfiletype <filetype>   Specify either trace, exported or plain
+ -loglevel <level>           Specify logging level e.g. off, info, fine
+ -reportpath <path>          Specify the path where report will be saved
+ -sourcedb <version>         Specify source database e.g. oracle_19c
+ -targetdb <version>         Specify target database e.g. epas_15
+ -version                    Print the version information and exit
+
+
+/usr/edb/sqlassess/bin/runSQLAssess.sh -version
+EDB SQL Assess ver 1.0.0-preview
+
+/usr/edb/sqlassess/bin/runSQLAssess.sh -appname TeleApp -inputfilepath /some/valid/path/ -inputfiletype plain -reportpath /some/valid/path/ -sourcedb Oracle_19c -targetdb epas_15 -loglevel FINE
+
+EDB SQL Assess ver 1.0.0-preview
+Input file type: PLAIN
+Input file path: /home/edb/Projects/
+Report path: /home/edb/Projects/
+Application Name: TeleApp
+Source Database: ORACLE_19C
+Target Database: EPAS_15
+Logging Level: FINE
+Processing File: t0.sql...
+Generating Report....
+Connected to H2 ..
+**********************
+Generating report ....
+**********************
+Report generation complete.
+Another example providing only the mandatory command line options.
+
+./runSQLAssess.sh -inputfilepath /home/edb/ -inputfiletype trace -reportpath /home/edb/
+EDB SQL Assess ver 1.0.0-preview
+Input file type: TRACE
+Input file path: /home/edb/
+Report path: /home/edb/
+Application Name: TestApp
+Source Database: ORACLE_19C
+Target Database: EPAS_15
+Logging Level: INFO
+Processing File: orcl_ora_117637.trc...
+Generating Report....
+Connected to H2 ..
+**********************
+Generating report ....
+**********************
+Report generation complete.
+```
+
+### `includeUIDs` parameter
+
+The `includeUIDs` parameter can filter queries from trace files with a comma separated list of Oracle UID values, like `106, 1120`. 
+
+For example, if a user provides the value `106`, then the queries associated only with UID 106 are assessed and all other queries are discarded. If a user doesn't provide a value for this parameter, by default the assessSQL utility discards queries for `UID=0` and assesses all other queries.
+
+To obtain the UID of an Oracle user, run the following query after connecting to the Oracle database as the user: 
+
+```sql
+SELECT UID FROM DUAL;
+```
+
+### `inputfiletype` and `inputfilepath` parameter
```

Comment:
I don't think we need this section on `inputfiletype` and `inputfilepath` parameters.  The text and note was relevant in the original beta release but no longer is.  As such, we can remove this section.